modal/partial_function.py

```diff
@@ -134,9 +142,8 @@ def _find_partial_methods_for_user_cls(
     return partial_functions
 
 
-def _find_callables_for_cls(user_cls: Type[Any], flags: _PartialFunctionFlags) -> Dict[str, Callable[..., Any]]:
-    """Grabs all method on a user class, and returns callables. Includes legacy methods."""
-    functions: Dict[str, Callable] = {}
+def _find_callables_for_cls(user_cls: Type[Any], flags: _PartialFunctionFlags) -> Dict[str, _PartialFunction]:
```

Comment:
Since this now returns PartialFunctions, it should probably be renamed (or just merged with `_find_partial_methods_for_user_cls`. It could be a bit confusing that `_find_callables_for_cls` returns PartialFunction whereas `_find_callables_for_obj` returns actual callable functions

Reply:
merged them!